### ISSUE TITLE
path and filename of pid_file defined in motion.conf but nothing happens

### ISSUE THREAD

**Issue by BirdiPiduino** [Motion-Project/motion#1604]
### Did you read the guide?

Yes

### What is the base version number of Motion being used?

4.5.x

### What was the install method?

Built from source code

### What is base architecture?

ARM-32bit

### What is the distro being used?

Raspbian

### Disto version number

"Raspbian GNU/Linux 11 (bullseye)"

### Camera type(s) being used?

PI cam via libcamerify

### Describe the issue/problem and steps to reproduce

Hi expert community,

I defined in motion.conf the full path and filename for pid_file as mentionned in documentation (I think).

```# File to store the process ID.```
```pid_file /home/pi/BirdRec/program/motion/pid_file```

I've tried with pid_file.txt.
I've tried with and without file pre-existing.

In this folder, I've located the motion.conf and it's works fine.

After starting motion, nothing happen. The pid_file isn't created and if it's pre-exist, it's stay empty.

In first, i want to thank you a lot for motion and your sharing.

BirdiPiDuino

### Relevant Motion log output (at log_level 8)

```shell
[0:motion] [NTC] [ALL] [Dec 11 12:20:13] main: Threads finished
[0:motion] [NTC] [ALL] [Dec 11 12:20:13] main: Motion terminating
[0:motion] [NTC] [ALL] [Dec 11 12:20:13] motion_remove_pid: Closing logfile (/home/pi/BirdRec/log/motion.log).
[0:motion] [NTC] [ALL] [Dec 11 12:20:16] motion_startup: Motion 4.5.0+git20221119-ec03434 Started
[0:motion] [NTC] [ALL] [Dec 11 12:20:16] motion_startup: Using log type (ALL) log level (NTC)
[0:motion] [NTC] [STR] [Dec 11 12:20:16] webu_start_strm: Starting all camera streams on port 8081
[0:motion] [NTC] [STR] [Dec 11 12:20:16] webu_strm_ntc: Started camera 0 stream on port 8081
[0:motion] [NTC] [STR] [Dec 11 12:20:16] webu_start_ctrl: Starting webcontrol on port 8080
[0:motion] [NTC] [STR] [Dec 11 12:20:16] webu_start_ctrl: Started webcontrol on port 8080
[0:motion] [NTC] [ENC] [Dec 11 12:20:16] ffmpeg_global_init: ffmpeg libavcodec version 58.91.100 libavformat version 58.45.100
[0:motion] [NTC] [ALL] [Dec 11 12:20:16] translate_init: Language: English
[0:motion] [NTC] [ALL] [Dec 11 12:20:16] motion_start_thread: Camera ID: 0 is from /home/pi/BirdRec/program/motion/motion.conf
[0:motion] [NTC] [ALL] [Dec 11 12:20:16] motion_start_thread: Camera ID: 0 Camera Name: BirdRec Device: /dev/video0
[0:motion] [NTC] [ALL] [Dec 11 12:20:16] main: Waiting for threads to finish, pid: 2634
[1:ml1:BirdRec] [NTC] [ALL] [Dec 11 12:20:16] motion_init: Camera 0 started: motion detection Enabled
[1:ml1:BirdRec] [NTC] [VID] [Dec 11 12:20:16] vid_start: Opening V4L2 device
[1:ml1:BirdRec] [NTC] [VID] [Dec 11 12:20:16] v4l2_device_open: Using videodevice /dev/video0 and input -1
[1:ml1:BirdRec] [NTC] [VID] [Dec 11 12:20:16] v4l2_device_capability: - VIDEO_CAPTURE
[1:ml1:BirdRec] [NTC] [VID] [Dec 11 12:20:16] v4l2_device_capability: - STREAMING
[1:ml1:BirdRec] [NTC] [VID] [Dec 11 12:20:16] v4l2_input_select: Name = "/base/soc/i2c0mux/i2c@1/imx219@"- CAMERA
[1:ml1:BirdRec] [NTC] [VID] [Dec 11 12:20:16] v4l2_norm_select: Device does not support specifying PAL/NTSC norm
[1:ml1:BirdRec] [NTC] [VID] [Dec 11 12:20:16] v4l2_pixfmt_try: Testing palette YU12 (640x480)
[1:ml1:BirdRec] [NTC] [VID] [Dec 11 12:20:16] v4l2_pixfmt_set: Using palette YU12 (640x480)
[1:ml1:BirdRec] [ERR] [VID] [Dec 11 12:20:16] v4l2_fps_set: Error setting fps. Return code -1
[1:ml1:BirdRec] [NTC] [ALL] [Dec 11 12:20:16] image_ring_resize: Resizing pre_capture buffer to 1 items
[1:ml1:BirdRec] [NTC] [ALL] [Dec 11 12:20:17] image_ring_resize: Resizing pre_capture buffer to 12 items
```

**Comment by Mr-Dave** [comment]
Thanks for reporting the issue. 

If ```daemon``` is set to off, it does not create it. (undocumented).  Before systemd, this partially made sense but now it is definitely an issue since we now set ```daemon off``` when running as a service.

**Comment by Mr-Dave** [comment]
closed via dc107bb16aae642a28ef4cc5980778c8ff977971
